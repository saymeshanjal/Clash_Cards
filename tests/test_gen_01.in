100
4 342 2
6 428 1
10 445 3
6 21 5
12 339 2
4 136 5
12 242 5
4 143 4
10 73 3
2 167 1
2 382 3
6 363 5
8 118 3
2 323 3
4 177 2
2 98 3
8 445 4
4 56 2
8 219 5
12 497 5
14 324 1
6 179 1
16 213 2
6 86 1
10 336 2
8 388 1
4 185 5
14 141 2
16 257 3
4 320 5
14 411 2
16 117 1
14 120 2
4 110 1
2 109 5
6 201 1
2 275 3
14 320 5
12 225 2
6 63 4
12 1 5
12 167 4
12 177 5
2 141 4
16 128 3
8 317 5
8 304 3
10 27 1
12 43 1
8 326 1
6 116 5
2 239 3
2 366 3
4 463 2
10 110 5
14 397 1
6 202 5
4 459 1
6 60 3
10 256 5
16 366 1
4 46 4
6 163 2
2 88 5
10 426 2
16 284 4
6 364 1
16 186 2
6 72 2
6 246 5
4 17 3
10 134 2
2 18 3
6 91 3
8 242 3
4 344 1
4 213 2
12 401 2
4 180 2
10 93 4
16 468 3
14 163 4
2 421 5
14 350 3
4 415 2
12 472 1
2 416 3
10 450 3
6 278 3
14 208 4
6 249 5
16 11 2
12 289 5
16 498 1
12 218 1
12 129 5
10 261 1
14 477 3
6 455 1
4 275 4