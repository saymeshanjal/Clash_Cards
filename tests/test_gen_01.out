1 2 3
10 10 0
110 142 0
1 19 0
286 287 351
0 6 0
219 705 0
0 5 1
110 142 0
1 1 0
0 2 0
1 19 0
27 43 0
0 2 0
1 2 3
0 2 0
8 42 20
1 2 3
8 62 0
219 705 0
1716 1716 0
10 10 0
4081 4082 4707
10 10 0
76 77 99
35 35 0
0 6 0
1078 1079 1275
6307 6563 0
0 6 0
1078 1079 1275
6435 6435 0
1078 1079 1275
3 3 0
0 2 0
10 10 0
0 2 0
987 2445 0
286 287 351
1 14 5
0 924 0
208 441 275
219 705 0
0 2 0
6307 6563 0
8 62 0
27 43 0
125 127 0
462 462 0
35 35 0
1 19 0
0 2 0
0 2 0
1 2 3
45 207 0
1716 1716 0
1 19 0
3 3 0
6 14 0
45 207 0
6435 6435 0
0 5 1
5 6 9
0 2 0
76 77 99
3809 5406 3655
10 10 0
4081 4082 4707
5 6 9
1 19 0
1 5 0
76 77 99
0 2 0
6 14 0
27 43 0
3 3 0
1 2 3
286 287 351
1 2 3
44 133 75
6307 6563 0
910 1520 1002
0 2 0
1652 1780 0
1 2 3
462 462 0
0 2 0
110 142 0
6 14 0
910 1520 1002
1 19 0
572 2804 9494
219 705 0
6435 6435 0
462 462 0
219 705 0
126 126 0
1652 1780 0
10 10 0
0 5 1
